docs/source/man.rst

```diff
@@ -471,11 +471,15 @@ Viewing the Detail Report
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 The ``qpc report detail`` command generates a report that contains the unprocessed facts that are gathered during a scan. These facts are the raw output from network, vcenter, and satellite scans.
 
-**qpc report detail --id** *scan_job_identifier* **(--json|--csv)** **--output-file** *PATH*
+**qpc report detail (--scan-job** *scan_job_identifier*|**--report** *report_identifier* **) (--json|--csv)** **--output-file** *PATH*
 
-``--id=scan_job_identifier``
+``--scan-job=scan_job_identifier``
+
+  Required. Contains the scan job identifier for the scan that is used to generate the report. Mutually exclusive with the ``--report`` option.
+
+``--report=report_identifier``
 
-  Required. Contains the scan job identifier for the scan that is used to generate the report.
+  Required. Contains the report identifier to retrieve.  Mutually exclusive with the ``--scan-job`` option.
```

Comment:
Possibly remove `Required` from lines 478 & 482. In other places in the man.rst where we have mutually exclusive groups we don't put required (ie. the --json/--csv for report or viewing scan jobs with --name/--id)

Reply:
Great suggestion!